app/src/main/java/com/asfoundation/wallet/viewmodel/TransactionsViewModel.java

```diff
@@ -105,6 +111,22 @@ public void prepare() {
     disposables.add(transactionViewInteract.hasPromotionUpdate()
         .subscribeOn(Schedulers.io())
         .subscribe(showNotification::postValue, this::onError));
+    disposables.add(transactionViewInteract.getUserLevel()
+        .subscribeOn(Schedulers.io())
+        .flatMap(userLevel -> transactionViewInteract.findWallet()
+            .subscribeOn(Schedulers.io())
+            .map(wallet -> {
+              boolean showSupport = userLevel == UserStats.MAX_LEVEL;
+              Log.e("TEST", "SHOW SUPPORT: " + showSupport);
```

Comment:
You must have forgotten this.
we can do it with:

 if (userLevel == UserStats.MAX_LEVEL) {
                registerSupportUser(wallet.address);
              } else {
                logoutSupportUser();
              }

Reply:
Agree

Reply:
Agree

Reply:
It was for testing purposes. I'm glad you did not complain about the log.